helper-cli/src/main/kotlin/commands/VerifySourceArtifactCurationsCommand.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.CommandWithHelp
+import com.here.ort.analyzer.HTTP_CACHE_PATH
+import com.here.ort.model.PackageCuration
+import com.here.ort.model.readValue
+import com.here.ort.utils.OkHttpClientHelper
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.collectMessagesAsString
+import com.here.ort.utils.hash
+
+import java.io.File
+import java.io.IOException
+
+import okhttp3.Request
+
+import okio.buffer
+import okio.sink
+
+@Parameters(
+    commandNames = ["verify-source-artifact-curations"],
+    commandDescription = "Verifies that all curated source artifacts can be downloaded and that the hashes are correct."
+)
+internal class VerifySourceArtifactCurationsCommand : CommandWithHelp() {
+    @Parameter(
+        description = "A YAML file that contains package curation data.",
+        names = ["--package-curations-file"],
+        order = PARAMETER_ORDER_MANDATORY
+    )
+    private lateinit var packageCurationsFile: File
+
+    override fun runCommand(jc: JCommander): Int {
+        var success = true
+        val curations = packageCurationsFile.readValue<List<PackageCuration>>()
+
+        curations.forEach { curation ->
+            curation.data.sourceArtifact?.let { sourceArtifact ->
```

Comment:
Please improve the readability a bit by separating things a bit,e.g. :

1. getSourceArtifactCurations
2. verify source artifact curations
3. print results